lib/Validator/Constraints/UniqueLanguageCodeValidator.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Validator\Constraints;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\LanguageService;
+use EzSystems\RepositoryForms\Data\Language\LanguageUpdateData;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+/**
+ * Will check if ContentType identifier is not already used in the content repository.
+ */
+class UniqueLanguageCodeValidator extends ConstraintValidator
+{
+    /**
+     * @var LanguageService
+     */
+    private $languageService;
+
+    public function __construct(LanguageService $languageService)
+    {
+        $this->languageService = $languageService;
+    }
+
+    /**
+     * Checks if the passed value is valid.
+     *
+     * @param LanguageUpdateData $value The value that should be validated
+     * @param Constraint|UniqueFieldDefinitionIdentifier $constraint The constraint for the validation
+     *
+     * @api
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        // REVIEW NOTE: Should this constraint not be checked on creation also? (if so, same with section)
```

Comment:
Can't really check instanceof trait, so changed to check instance of API struct, alternative is to check if one of the data classes.